docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_advanced_arming:
+
+***********************************************
+Advanced Arming
+***********************************************
+Vertiq modules can support an advanced arming feature, allowing the user to control the armed state of the module with throttle commands or manually 
+and to configure specific behaviors to occur at armed state transitions. Modules will not react to throttle messages until they have armed, providing improved safety. 

```

Comment:
saying improved safety seems like more of an advertisement statement more than something that should be in a manual.